$NetBSD: patch-ab,v 1.3 2013/04/13 19:07:21 dholland Exp $

- Fix C syntax (string constants continued onto the next line must
have a backslash)

--- src/Usage.h.orig	2002-03-26 16:55:20.000000000 +0100
+++ src/Usage.h
@@ -1,13 +1,13 @@
 void Usage()
 {
 printf(
-"Usage: ./cygne romfile options
- Where options are:
-
- romfile           : Filename of the romfile. It can have several
-                     extensions (.ws/.wsc/.rom/.bin). Gzipped and
-                     zipped (.gz and .zip) are also supported.
- -double           : Scale the screen 2 times the normal size.
- -scanline         : Allows scanline output with scaling.
+"Usage: ./cygne romfile options\n\
+ Where options are:\n\
+\n\
+ romfile           : Filename of the romfile. It can have several\n\
+                     extensions (.ws/.wsc/.rom/.bin). Gzipped and\n\
+                     zipped (.gz and .zip) are also supported.\n\
+ -double           : Scale the screen 2 times the normal size.\n\
+ -scanline         : Allows scanline output with scaling.\n\
 ");
 }
